package.json

```
@@ -9,7 +9,7 @@
     "compile-library": "webpack --output-library CommonsValidator --output-library-target var dist bin/commons-validator-js.src.js",
     "prepublish": "npm test && npm run compile && npm run compile-library",
     "build": "babel bin/commons-validator-js.src.js | uglifyjs > bin/commons-validator-js.js",
-    "release": "npm install wnpm-ci && wnpm-release -- --no-shrinkwrap && npm install bower-auto-release && bower-auto-release --dist bin"
+    "release": "wnpm-release -- --no-shrinkwrap && bower-auto-release --dist bin"
```

That's should solve the problem, npx fetch the latest package & execute it

```suggestion
    "release": "npx wnpm-release -- --no-shrinkwrap && npx bower-auto-release --dist bin"
```